docs/docs-content/getting-started/vmware/vmware.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_label: "Deploy a Cluster to VMware"
```

Comment:
The VMware card is not present on the getting started index page :)

Reply:
Thanks for spotting that! Added!